addons/mail/static/src/discuss/thread_model_patch.js

```diff
@@ -15,7 +15,8 @@ patch(Thread.prototype, "discuss/core/common", {
             );
         }
         if (this.type === "chat") {
-            return `/web/image/res.partner/${this.chatPartnerId}/avatar_128`;
+            const persona = this._store.personas[this.chatPartnerId];
```

Comment:
The correct way to write the persona `localId` in 16.3 is `this._store.personas[createLocalId("partner", this.chatPartnerId)]`

It is used several times in the context of thread, it would be nice to create a getter `chatPartner` directly.
This will also make the forward-port somewhat "compatible" with code in master.